src/components/Media/Iframe/Iframe.tsx

```diff
@@ -11,21 +13,91 @@ const b = block('media-component-iframe');
 
 const Iframe = (props: MediaComponentIframeProps) => {
     const {iframe, margins = true} = props;
-    const {height = 400, src, width, name, title} = iframe;
+    const {height = 400, src, width = '100%', name, title, justifyContent = 'center'} = iframe;
+
+    const formContainerRef = useRef<HTMLDivElement>(null);
+    const iframeRef = useRef<HTMLIFrameElement>();
+    const [iframeId] = useState(uuidv4());
+
+    const updateFormIframe = useCallback(
```

Comment:
why do we have form in Iframe component? I can't get the relation, seems like there should be no hard connection between forms and this iframe component

Reply:
Copy + Paste effect. Fixed